crates/oxc_ast/src/ast/js.rs

```diff
@@ -19,6 +19,7 @@ pub struct Program<'a> {
     pub span: Span,
     pub source_type: SourceType,
     pub directives: Vec<'a, Directive<'a>>,
+    pub interpreter: Option<Hashbang<'a>>,
```

Comment:
Shall we change the key to `shashbang` as well? I've got no idea what a `interpreter` (from babel). The spec just calls it a hashbang.

Reply:
yeah, I thought of that at the beginning but wasn't sure